CHANGELOG.md

```diff
@@ -18,6 +19,8 @@ This document outlines major changes between releases.
    set `S3_GW_WALLET_ADDRESS` instead of `S3_GW_ADDRESS`
    2. `.yaml` config: to set wallet path use `wallet.path` instead of `wallet`, 
    to set wallet address use `wallet.address` instead of `address`
+2. You can change default value `0` of `set_copies_number` in `.yaml` config by setting of `neofs.set_copies_number`, or
```

Comment:
I'm not sure we have to mention this. As I understand `Updating from ...` section is for breaking changes